design/account.go

```diff
@@ -212,7 +212,24 @@ var _ = a.Resource("users", func() {
 		a.Response(d.Unauthorized, JSONAPIErrors)
 		a.Response(d.Forbidden, JSONAPIErrors)
 		a.Response(d.Conflict, JSONAPIErrors)
+	})
 
+	a.Action("updateByServiceAccount", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.PATCH(":id"),
+		)
+		a.Description("update the user")
+		a.Params(func() {
+			a.Param("id", d.String, "id")
```

Comment:
Gets confusing so let's mention that it's the Identity ID.

Reply:
Well... wanted to be consistent with GET /api/user/:id. We can update the description though: a.Param("id", d.String, "identity ID")

Reply:
yea that's what I meant.